operations/agent-flow-mixin/dashboards/cluster-overview.libsonnet

```diff
@@ -0,0 +1,173 @@
+          Node count.
+        |||) +
+        panel.withPosition({ h: 9, w: 8, x: 0, y: 0 }) +
+        panel.withQueries([
+          panel.newInstantQuery(
+            expr='count(cluster_node_info)'
+          ),
+        ])
+      ),
+      // Node table
+      (
+        panel.new('Node table', 'table') +
+        panel.withDescription(|||
+          Nodes info.
+        |||) +
+        panel.withPosition({ h: 18, w: 16, x: 8, y: 0 }) +
+        panel.withQueries([
+          panel.newInstantQuery(
+            expr='cluster_node_info',
+            format='table',
+          ),
+        ]) +
+        panel.withTransformations([
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {
+                Time: true,
+                Value: false,
+                __name__: true,
+                cluster: true,
+                namespace: true,
+                state: false,
+              },
+              indexByName: {},
+              renameByName: {
+                Value: 'Dashboard',
+                instance: '',
+                state: '',
+              },
+            },
+          },
+        ]) +
+        panel.withOverrides(
+          [
+            {
+              matcher: {
+                id: 'byName',
+                options: 'Dashboard',
+              },
+              properties: [
+                {
+                  id: 'mappings',
+                  value: [
+                    {
+                      options: {
+                        '1': {
+                          index: 0,
+                          text: 'Link',
+                        },
+                      },
+                      type: 'value',
+                    },
+                  ],
+                },
+                {
+                  id: 'links',
+                  value: [
+                    {
+                      targetBlank: false,
+                      title: 'Detail dashboard for node',
+                      url: '/d/agent-cluster-node/grafana-agent-flow-cluster-node?var-instance=${__data.fields.instance}&var-datasource=${datasource}&var-loki_datasource=${loki_datasource}&var-cluster=${cluster}&var-namespace=${namespace}',
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        )
+      ),
+      // Convergance state
+      (
+        panel.new('Convergance state', 'stat') +
+        panel.withDescription(|||
+          "Whether the cluster state has converged.
+
+          It is normal for the cluster state to be diverged briefly as gossip events propagate. It is not normal for the cluster state to be diverged for a long period of time.
+
+          This will show one of the following:
+
+          * Converged: Nodes are aware of all other nodes, with the correct states.
+          * Not converged: A subset of nodes aren't aware of their peers, or don't have an updated view of peer states."
+        |||) +
```

Comment:
Ooh, after we merge this, I would love to work on a state history for a follow up PR so we can show how long the converged/not-converted state has been going on for.